### ISSUE TITLE
Emoji-named dashboards trigger redirect loop 🐴

### ISSUE THREAD

**Issue by KristianLyng** [grafana/grafana#22618]
**What happened**:

I made a wonderful dashboard, named it 🐴 and saved.

Then ended in a redirect loop, presumably because Grafana tries to redirect me to the proper name, but is confused by my horsing around.

**What you expected to happen**:

Horses! Lots of horses! (Seriously)

Also: to be able to see the dashboard without a redirect loop.

**How to reproduce it (as minimally and precisely as possible)**:

Make a dashboard named 🐴. I assume other emojis have similar problems, but it's fine by me if you exclusively fix this for horse-emojis.

**Anything else we need to know?**:

🐴

**Environment**:
- Grafana version: v6.6.0
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser: Chromium on Debian
- Grafana plugins:
- Others:🐴

**Comment by tskarhed** [comment]
This issue made my day.

**Comment by dprokop** [comment]
Thanks for reporting this, that's definitely something we might look at at some point;) I'm not really sure if emojis are supported in URLs, so let's maybe wait for some more evidence that this is needed. 

In the meantime 🐴🐴🐴

**Comment by aknuds1** [comment]
Perfect on a Friday! :D

**Comment by KristianLyng** [comment]
It is a sad day:

> Dashboard 🐴 deleted

As for URL-support, it's possible to hex-encode them and puny-code them just fine, e.g. http://xn--8o8h.friday.horse/ is valid for 🐴.friday.horse, and http://txt.friday.horse/?foo=%F0%9F%90%B4 is valid for http://txt.friday.horse/?foo=🐴 - the browser will do the translation.... So I'm actually guessing this issue might affect other special characters? I checked with my own surname, but that just converted the ø to an o, so that's seemingly fine? 

Also, it seems to only trigger if the original name is weird, e.g.: I tried making a dashboard named ß, and that got treated as "ss", but when I renamed it to ♥ and 🐴, it still got treated as ss?
